pkg/plugins/runtime/k8s/controllers/gatewayapi/gateway_controller.go

```diff
@@ -52,7 +52,7 @@ func (r *GatewayReconciler) Reconcile(ctx context.Context, req kube_ctrl.Request
 		if kube_apierrs.IsNotFound(err) {
 			// We don't know the mesh, but we don't need it to delete our
 			// object.
-			err := common.ReconcileLabelledObject(ctx, r.TypeRegistry, r.Client, req.NamespacedName, core_model.NoMesh, &mesh_proto.MeshGateway{}, nil)
+			err := common.ReconcileLabelledObject(ctx, r.Log, r.TypeRegistry, r.Client, req.NamespacedName, core_model.NoMesh, &mesh_proto.MeshGateway{}, nil)
```

Comment:
Does it make sense to pass in a log already init with `namespace` and `name` (and maybe type) as the `reconcile` log already almost logs this.

Reply:
I'm not sure. We always pass this data so I guess it's easier to use `ReconcileLabelledObject` if you can just pass the log without extra line?

Reply:
Ok not strong opinion here and you are the owner here :)